.github/workflows/build.yaml

```diff
@@ -0,0 +1,41 @@
+# This is a reusable workflow that is called by other
+# Actions workflows.  It checks, tests, and builds the package.
+
+name: Build the package
+on:
+  workflow_call:
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: Check Python ${{ matrix.python }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Should we leave a comment that it’s recommended to [pin a Runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) version specifically?

Reply:
See commit 8db20dd4e6538c089f77e3aa9c4dcbf0b53ee422.